apps/frontend/src/pages/auth/libs/maps/auth-route-to-header.map.ts

```diff
@@ -0,0 +1,8 @@
+const authRouteToHeader: Record<string, string> = {
+	"/forgot-password": "forgot password",
+	"/reset-password": "reset password",
+	"/sign-in": "sign in",
+	"/sign-up": "sign up",
+};
```

Comment:
```suggestion
const authRouteToHeader: Record<ValueOf<typeof AppRoute>>, string> = {
	[AppRoute.FORGOT_PASSWORD]: "forgot password",
	"/reset-password": "reset password",
	"/sign-in": "sign in",
	"/sign-up": "sign up",
};
```

Reply:
We use pathname as an index provided to this map, it is always of type string